# Button name to buttons index mapping
/bt_controller:
  ros__parameters:
        A: 0
        B: 1
        X: 2
        Y: 3
        LB: 4
        RB: 5
        HOME: 6
        MENU: 7
        LEFT_JOY_PRESS: 8
        RIGHT_JOY_PRESS: 9

        # Control name to axis index mapping
        LEFT_JOY_LR: 0
        LEFT_JOY_UD: 1
        RIGHT_JOY_LR: 2
        RIGHT_JOY_UD: 3
        RT: 4
        LT: 5
        D_PAD_LR: 6
        D_PAD_UD: 7
/joy:
  ros__parameters:
        dev: /dev/input/js4
